### ISSUE TITLE
Dub package custom init type

### ISSUE THREAD

**Issue by andre2007** [skoppe/spasm#8]
Dub will get the feature to have custom init types.
I saw spasm can make use if this (bootstrapping)
Could you have a look whether this functionality is useful for you?

https://github.com/dlang/dub/pull/1658

**Comment by skoppe** [comment]
It would be awesome! I already kept a close eye on it ;)

**Comment by andre2007** [comment]
@skoppe support for custom unit types was merged. Did you have a chance to look wheter it is useful for you?

**Comment by skoppe** [comment]
Yeah, I saw it when it got merged. It is very useful for users of spasm.

There are several more important matters that I need to focus on right now, so this will have to wait.

Thank you for reminding me about it. I will leave this issue open so not to forget.
